### DISCUSSION TITLE
Merging Classes

### DISCUSSION THREAD

**Discussion by lokmanm** [joe-bell/cva#101] [Ideas]
Hi Joe and other contributors.

First of all thanks for this package as we use it extensively with our projects. 

Is there any plan on adding the possibility of merging classes, kinda similar to how [tailwind-merge](https://github.com/dcastil/tailwind-merge) does it?

It will be very helpful when working with utility classes to be able to merge the final class.

Thanks again.

**Comment by joe-bell** [comment]
Not just yet, but here’s an example of how to achieve it in the meantime:

https://github.com/joe-bell/cva#handling-style-conflicts
